rhemyn/compile.js

```diff
@@ -122,6 +123,18 @@ const wrapQuantifier = (node, method, get, stringSize) => {
     ...number(0, Valtype.i32),
     [Opcodes.local_set, QuantifierTmp],
 
+	// if len - counter == 0, if min == 0, succeed, else fail
```

Comment:
I am likely missing something but just to check, do we really need a bounds check in each of these functions since `counter` is only updated before the existing check (excluding the removed/commented out section?)? I understand running with an empty string is broken but not sure if that is just an edge-case. ty!

Reply:
> we can't just add a little check at the top to see if the length is equal to zero and auto fail or succeed

Why not? I understand the rest, thanks that makes sense.

Reply:
I think in hindsight that might be the best option, I'll have a go at implementing something like that